src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -74,19 +74,25 @@ private void testCreateInstructorAction() throws Exception {
         InstructorAttributes instructor = new InstructorAttributes();
         
         String shortName = "Instrúctör";
-        instructor.name = "AHPUiT Instrúctör";
-        instructor.email = "AHPUiT.instr1!@gmail.tmt";
+        instructor.name = "AHPUiT Instrúctör+++";
+        instructor.email = "AHPUiT+++_.instr1!@gmail.tmt";
         String institute = "TEAMMATES Test Institute 1";
-        String demoCourseId = "AHPUiT.instr1_.gma-demo";
+        String demoCourseId = "AHPUiT____.instr1_.gma-demo";
         
         String instructorDetails = instructor.name + " | " + instructor.email + "\n"
                                  + instructor.name + " | " + instructor.email + " | " + institute;
         
-        ______TS("action fail & success: add multiple instructors");
+        ______TS("action success: instructor details are sent across to the backend properly");
         BackDoor.deleteAccount(TestProperties.TEST_INSTRUCTOR_ACCOUNT);
         BackDoor.deleteCourse(demoCourseId);
         BackDoor.deleteInstructor(demoCourseId, instructor.email);
         homePage.createInstructorByInstructorDetailsSingleLineForm(instructorDetails);
+        InstructorAttributes instructorInBackend = BackDoor.getInstructorByEmail(instructor.email, demoCourseId);
+        
+        assertEquals(instructor.getName(), instructorInBackend.getName());
+        assertEquals(instructor.getEmail(), instructorInBackend.getEmail());
+        
+        ______TS("action fail & success: add multiple instructors");
```

Comment:
And this as well. This is why my previous suggestion was to move the two `assertEquals`. To be more precise, to between the `assertEquals("Instructor AHPUiT Instructor+++...` and `homePage.clearInstructorDetailsSingleLineForm();`.